manifests/candlepin.pp

```diff
@@ -49,6 +70,13 @@
 
   if $deploy {
 
+    Cert[$tomcat_cert_name] ~>
```

Comment:
A quick spec test tells me that just for a requires this works. The difference is that here's a notify. I'm not sure if a notify is needed, but IMHO we should remove all these relations and add autonotify to the type if it's needed.